api-docs/paths/projects/user-feedback.json

```diff
@@ -1,7 +1,7 @@
 {
   "get": {
     "tags": ["Projects"],
-    "description": "Return a list of user feedback items within this project.",
+    "description": "Return a list of user feedback items within this project.\n\n*This list does not include submissions from the [User Feedback Widget](https://docs.sentry.io/product/user-feedback/#user-feedback-widget). This is because it is based on an older format called User Reports - read more [here](https://develop.sentry.dev/application/feedback-architecture/#user-reports).*",
```

Comment:
was this manually modified?👀

Reply:
Yeah - turns out the json files is what api-docs actually uses. But they want the docstrings to stay consistent since they might switch to using those in the future.

Reply:
oh cool i did not know - TIL!